kickstarts/almalinux-8-base.ks

```diff
@@ -1,7 +1,9 @@
 # AlmaLinux 8 kickstart file for x86_64 base Docker image
 
 # install
-url --url https://repo.almalinux.org/almalinux/8.5/BaseOS/$basearch/os/
+url --url https://repo.almalinux.org/almalinux/8.5/BaseOS/$basearch/kickstart/
```

Comment:
Kickstart URL populated on initial release, regular `os` gets the incremental updates. any missing packages using just one URL?

Reply:
The installer can't find AppStream's packages on ppc64le. @andrewlukoshko and I tried to find the root of the issue. Nothing unusual has been observed on the repository of `ppc64le` :thinking: